gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -16,8 +16,45 @@
 
 import apache_beam as beam
 
-from gcp_variant_transforms.libs import vcf_field_conflict_resolver
-from gcp_variant_transforms.libs.variant_merge import header_merge_util
+from gcp_variant_transforms.beam_io import vcf_header_io
+from gcp_variant_transforms.libs import vcf_field_conflict_resolver, header_merger
```

Comment:
nit: Please only import one module in one line. One exception is for typing import. For instance,`from typing import Any, Dict`.

Reply:
Done.